ScheduleCreator/DBAdapter.java

```diff
@@ -19,27 +26,26 @@
  * 
  * Last Updated: 3/3/2020
  */
-public class DBAdapter {
-
-    protected static File selectedCourseFile = new File("src/ScheduleCreator/resources/raw/user_selected_courses.txt");
-
-    ;
+public class DBAdapter { 
 
     /**
      * Saves the selected course (abbreviation and number) and saves to database.
      * @param _course
      * @throws Exception
      */
+    
+    public DBAdapter() {
+    }
 
-    public static void saveCourse(String _course) throws Exception {
-
-        //Open file to add new classes.
-        FileWriter output = new FileWriter(selectedCourseFile, true);
-
-        //Adds new selected course to new line.
-        output.append(_course + "\n");
-
-        output.close();
+    public  void saveCourse(String _course) throws Exception {
+//
+//        String file = getClass().getResource("resources/raw/spring2020").getFile();
+//        PrintWriter output = new PrintWriter(new BufferedWriter(new FileWriter(file)));
+//
+//        //Adds new selected course to new line.
+//        output.append(_course + "\n");
+//
+//        output.close();
```

Comment:
`getClass().getResource()` is exactly what I was looking for. 

What does this method do? We shouldn't write to files in the classpath.

I'm guessing getClass().getResource() doesn't work from CoursesController because it's in the wrong dir?

Reply:
You can disregard all of the code that's commented out. 

getResource() returns a URL object and I was just trying to figure out how to work with that, but I didn't get anywhere.

Reply:
> You can disregard all of the code that's commented out.

Oh, duh.